Ambit/Config/FilterPlugin.ini

```diff
@@ -0,0 +1,9 @@
+[FilterPlugin]
+; This section lists additional files which will be packaged along with your plugin. Paths should be listed relative to the root plugin directory, and
+; may include "...", "*", and "?" wildcards to match directories, files, and individual characters respectively.
+;
+; Examples:
+;    /README.txt
+;    /Extras/...
+;    /Binaries/ThirdParty/*.dll
+AWSAmbit.hda
```

Comment:
Shouldn't this be `/HoudiniDigitalAssets/AWSAmbit.hda` or other variant with wildcard?

Reply:
Nope. Included in this PR is a move of that .hda up a level to simplify packaging. There is no "HoudiniDigitalAssets" folder anymore. Also, I made the conscious decision not to use a wildcard for now in favor of making inclusion of this and any future HDAs in the packaged build explicit. We can always change that decision later if the need arises.